sdk/resourcemover/Azure.ResourceManager.ResourceMover/src/autorest.md

```diff
@@ -160,4 +176,6 @@ directive:
     where: $.paths..parameters[?(@.name === 'moveResourceName')]
     transform: >
       $['x-ms-client-name'] = 'moverResourceName';
+  - remove-operation: MoveCollections_Delete
```

Comment:
This delete operation can't work because of the Azure_AsyncOperation header difference between different resposne, created an issue to track https://github.com/Azure/azure-sdk-for-net/issues/30952